packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -389,19 +432,24 @@ export default class LocusInfo extends EventsScope {
       }
       this.host = host;
     }
+    else {
+      this.compareAndUpdateFlags.compareSelfAndHost = false;
+    }
   }
 
-  updateMeetingInfo(info) {
-    if (info && !isEqual(this.info, info)) {
-      const parsedInfo = InfoUtils.getInfos(this.info, info);
+  updateMeetingInfo(info, self) {
+    if (info && (!isEqual(this.info, info) ||
+    !isEqual(InfoUtils.parsePolicy(info), InfoUtils.parsePolicy(this.info)) ||
```

Comment:
@chrisadubois Should this be a parent check !isEqual(this.info, info)
And if its true then we can do the other checks ' !isEqual(InfoUtils.parsePolicy(info), InfoUtils.parsePolicy(this.info)) '

Reply:
as discussed, no